lua/upbc.cc

```diff
@@ -61,7 +61,7 @@ static void PrintString(int max_cols, absl::string_view* str,
     } else if (ch == '\'') {
       printer->PrintRaw("\\'");
       max_cols--;
-    } else if (isprint(ch)) {
+    } else if (isprint(static_cast<int>(static_cast<unsigned char>(ch)))) {
```

Comment:
Can you make a helper for this, and do the <0 check explicitly instead of casting through:

```
static void IsPrint(int ch) {
  // isprint(ch) with negative values is UB.
  return ch < 0 ? false : isprint(ch);
}
```

Reply:
Thanks and fixed.